src/components/PeopleTable.tsx

```diff
@@ -1,52 +1,166 @@
-export const PeopleTable = () => {
+import classNames from 'classnames';
+import { useParams } from 'react-router-dom';
+
+import { Person } from '../types';
+import { PersonLink } from './personLink';
+import { SearchLink } from './SearchLink';
+import { SearchParams } from '../utils/searchHelper';
+import { SortTypes } from '../types/sortParams';
+
+type Props = {
+  setSearchWith:(params: SearchParams) => void,
+  people: Person[],
+  allPeople: Person[],
+  order: string,
+  sort: string,
+  searchParams: URLSearchParams,
+};
+
+export const PeopleTable: React.FC<Props> = ({
+  setSearchWith,
+  people,
+  allPeople,
+  order,
+  sort,
+  searchParams,
```

Comment:
no need to pass all these variables as props
Search params are available anywhere from hook